Source/WebCore/platform/audio/PlatformMediaSessionManager.cpp

```diff
@@ -661,6 +667,15 @@ Vector<WeakPtr<PlatformMediaSession>> PlatformMediaSessionManager::sessionsMatch
     return matchingSessions;
 }
 
+WeakPtr<PlatformMediaSession> PlatformMediaSessionManager::firstSessionMatching(const Function<bool(const PlatformMediaSession&)>& predicate) const
```

Comment:
I would be returning a RefPtr instead here

Reply:
Unfortunately, `PlatformMediaSession` isn't ref-counted.  :(